server/offenceCodeDecisions/DecisionTrees.test.ts

```diff
@@ -0,0 +1,44 @@
+import decisionTrees from './DecisionTrees'
+
+function findDuplicates<T>(toCheck: Array<T>) {
+  const unique = new Set(toCheck)
+  return toCheck
+    .filter(item => {
+      if (unique.has(item)) {
+        unique.delete(item)
+        return null
+      }
+      return item
+    })
+    .filter(item => item != null)
+}
+
+describe('decisions', () => {
+  it.skip('toString', () => {
+    // Not a test but useful output
+    // eslint-disable-next-line no-console
+    decisionTrees.forEach(decisionTree => console.log(decisionTree.toString()))
+  })
+
+  it.skip('no duplicate codes', () => {
```

Comment:
Why skipped? is it because codes can be duplicated, because i think that is correct.....

we have situations where an answer before the end doesn't actually change the output code (so the same branch), but that is information that is sent to the API.

Reply:
Until we do todos..